src/form-field/__tests__/form-field-control-id.test.tsx

```diff
@@ -59,6 +65,21 @@ describe('controlId', () => {
       expect(wrapper.findConstraint()?.getElement().id).toBe(`${formFieldControlId}-constraint`);
     });
 
+    test('sets the right ids for all props when everything including warningText is set at the sameTime', () => {
```

Comment:
maybe also add `warningText` to the test above and check that everything is still as expected?

Reply:
or if it doesn't make sense to have both, then rename the test above to include errorText

Reply:
I added the test separately because when errorText and warningText exist at the same time, warningText doesn't get shown. But I added it to the test above in the new commit and asserted that warningText becomes undefined. Also renamed the test too.